bin/adam-submit

```diff
@@ -22,40 +22,15 @@ SCRIPT_DIR="$(cd `dirname $0`/..; pwd)"
 
 # Get list of required jars for ADAM
 ADAM_JARS=$("$SCRIPT_DIR"/bin/compute-adam-jars.sh)
+ADAM_CLI_JAR=$(echo "$ADAM_JARS" | rev | cut -d',' -f1 | rev )
```

Comment:
Blindly copied from http://stackoverflow.com/questions/22727107/how-to-find-the-last-field-using-cut-linux

```bash
data=foo,bar,baz,qux
last=${data##*,}
```